ReSharper.FSharp/src/FSharp.Psi.Daemon/src/Stages/InferredTypeCodeVisionProvider.fs

```diff
@@ -27,21 +28,22 @@ open JetBrains.Util
 
 module FSharpInferredTypeHighlighting =
     let [<Literal>] Id = "CodeInsights"
-    let [<Literal>] ProviderId = "F# Inferred types"
+    let ProviderId = Strings.FSharpInferredTypeHighlighting_ProviderId
```

Comment:
Just a nit-pick, not a serious issue: module values normally don't start with uppercase. Uppercase values are usually the ones that are available in pattern matching (literals, union cases, exceptions).